### PR TITLE
[AGENT-4682] Implement Airflow Operators for Training a specific blueprint 

### PR DESCRIPTION
**Pull request by oleksandr-saienko**
# This repository is public. Do not put here any private DataRobot or customer's data: code, datasets, model artifacts, .etc.

## Summary
Added TrainModelOperator and ModelTrainingJobSensor for model training using specific blueprint
## Rationale
users should be able to train a model using specific blueprint

### CODE REVIEW

**Comment by oleksandr-saienko** [comment]
Jenkins run

**Comment by oleksandr-saienko** [comment]
Jenkins run

**Comment by oleksandr-saienko** [comment]
Jenkins run

**Review by dulcardo — APPROVED** [review]

File: datarobot_provider/sensors/model_insights.py [hunk 1]
```diff
@@ -47,7 +49,11 @@ def __init__(
 
         self.hook = DataRobotHook(datarobot_conn_id)
 
-    def poke(self, context: Dict[Any, Any]) -> bool:
+    def get_job_result(self, context: Dict[Any, Any]) -> bool:
+        # Default implementation return True if job is completed:
+        return True
```
Comment (elatt) [inline datarobot_provider/sensors/model_insights.py:54]: Should this be abstract? I'm not familiar enough with the DR api to know all the nuances but it seems weird to have a default implementation that just returns true?

**Review by elatt — APPROVED** [review]

File: datarobot_provider/sensors/model_insights.py (hunk 1 above)
Comment (oleksandr-saienko) [reply datarobot_provider/sensors/model_insights.py:54]: All good, some jobs dont need return any object, just job complete status, so we can use DataRobotJobSensor directly (we just need return True when job is complete), but some jobs returns a result object after job is complete in this case we need to use corresponding child class (like ModelTrainingJobSensor this case) and return PokeReturnValue which contains job status and some job result object (in this case PokeReturnValue(True, trained_model.id))

**Pull request merged**